### ISSUE TITLE
Users have requested the ability to set dimmer switches to fans

### ISSUE THREAD

**Issue by w1ll1am23** [python-wink/python-wink#112]
@vickyg3 I know you added support for the official GE fan controllers, but it looks like some users are using non-fan controller (standard dimmers) as fan controllers. Thoughts on adding support for user specified devices not just based on model? 

For reference this is most recently being discussed here https://www.reddit.com/r/homeassistant/comments/9d6f5c/hassio_wink_addon_change_switch_to_fan/

**Comment by vickyg3** [comment]
Sure. I think the following has to be done:
* Adding some kind of an API function to python-wink to pass in the devices which you want recognized as fans.
* And a configuration entry for the wink component in hass which will make use of the added python-wink API.

**Comment by w1ll1am23** [comment]
Correct, the problem I see is how should the user specify which devices should be fans? I would think it would have to be based on the object_id but that isn't exposed to the HA user. So it would really need to be based on name I guess?
